src/verify.c

```diff
@@ -557,6 +560,13 @@ static bool parse_opt(int opt, char *optarg)
 		}
 		cmdline_option_picky_whitelist = optarg;
 		return true;
+	case 'B':
+		if (!optarg) {
+			fprintf(stderr, "Invalid --bundle argument\n\n");
```

Comment:
The other options of the _verify_ command have a text like this:

> "Missing --picky-tree argument\n\n"

https://github.com/clearlinux/swupd-client/blob/master/src/verify.c#L544

So I'd prefer if this message stays consistent with those messages by changing "Invalid" with "Missing"

Reply:
Updating.

Reply:
Actually I'm curious about this. This error should never happen as getopt catches this condition and raises its own error. Is there a reason we have these checks? Or am I misreading the man page?

Reply:
you're right. this check doesn't make sense. You can remove that.

Reply:
Alrighty pushing an update.